docs/filter_kgraph_top_n.md

```diff
@@ -0,0 +1,59 @@
+# filter kgraph (top n)
+
+_subclass of [filter_kgraph](./filter_kgraph.md)_
+
+This operation removes kgraph edges that have attribute values are below/above top/bottom n values.
+
+### examples
+
+- [input](../examples/fill_and_filter/messages/05_filtered_kgraph_stat_input.json), [output](../examples/fill_and_filter/messages/08_filtered_kgraph_top_n_output.json)
+
+### input requirements
+
+None
+
+### output guarantees
+
+- no edges in the kgraph which have attribute values are below/above the mean +/- n standard deviations.
+
+### allowed changes
+
+- remove knodes
+- remove kedges
+
+### parameters
+
+```yaml
+properties:
+  edge_attribute:
+    description: The name of the edge attribute to filter on.
+    example: normalized_google_distance
+    type: string
+  qnode_key:
+    default: null
+    description: If remove_connected_nodes is set to True this indicates if you only
+      want nodes corresponding to a specific qnode_key to be removed. If not provided
+      the qnode_key will not be considered when filtering.
+    example: n01
+    type: string
+  remove_connected_nodes:
+    default: false
+    description: Indicates whether or not to remove the nodes connected to the edges.
+    example: true
+    type: bool
+  threshold:
```

Comment:
`max_results` instead of threshold

Reply:
marking as resolved since this has been changed